app/models/sushi/platform_report.rb

```diff
@@ -67,28 +71,19 @@ def attribute_performance_for_resource_types
       data_for_resource_types.group_by(&:resource_type).map do |resource_type, records|
         { "Data_Type" => resource_type || "",
           "Access_Method" => "Regular",
-          "Performance" => {
-            "Total_Item_Investigations" =>
-            records.each_with_object({}) do |record, hash|
-              hash[record.year_month.strftime("%Y-%m")] = record.total_item_investigations
-              hash
-            end,
-            "Total_Item_Requests" =>
-            records.each_with_object({}) do |record, hash|
-              hash[record.year_month.strftime("%Y-%m")] = record.total_item_requests
-              hash
-            end,
-            "Unique_Item_Investigations" =>
-            records.each_with_object({}) do |record, hash|
-              hash[record.year_month.strftime("%Y-%m")] = record.unique_item_investigations
-              hash
-            end,
-            "Unique_Item_Requests" =>
-            records.each_with_object({}) do |record, hash|
-              hash[record.year_month.strftime("%Y-%m")] = record.unique_item_requests
-              hash
-            end
-          } }
+          "Performance" => performance(records, resource_type) }
+      end
+    end
+
+    def performance(records, resource_type)
+      metric_types.each_with_object({}) do |metric_type, hash|
+        # Skip "Unique_Title_Requests" & "Unique_Title_Investigations" for all resource_types besides books
+        next if !resource_type.casecmp("book").zero? && (metric_type == "Unique_Title_Requests" || metric_type == "Unique_Title_Investigations")
```

Comment:
I see that we're skipping everything but `"book"`, could you comment why that is the case?

Reply:
> The Unique_Title metrics MUST NOT be used for Data_Types other than Book and Reference_Work as they are not meaningful for them. If a title contains both Open and Controlled sections or sections with different YOPs, the usage must be broken down by Access_Type and YOP so that the total counts are consistent between reports including and not including these columns/elements.

My read on the requirements is that we need to consider other resource_type; and I also suspect based on my read, that the unique may not be unique for a given day, but unique for a given person.

>  Unique_Title metrics were introduced in R5 to help normalize eBook metrics, and are retained in R5.1. Unique_Title metrics are only increased by 1 no matter how many (or how many times) chapters or sections are accessed in a given user session. Unique_Title metrics provide comparable eBook metrics regardless of the nature of the platform and how eBook content is delivered. They are comparable across report providers and across releases.